parm/land/obs/config/adpsfc_snow.yaml

```diff
@@ -6,11 +6,11 @@ obs space:
   obsdatain:
     engine:
       type: H5File
-      obsfile: $(DATA)/obs/$(OPREFIX)adpsfc_snow_{{ current_cycle | to_YMDH }}.nc4
+      obsfile: $(DATA)/obs/$(OPREFIX)adpsfc_snow.nc4
   obsdataout:
     engine:
       type: H5File
-      obsfile: $(DATA)/diags/diag_adpsfc_snow_{{ current_cycle | to_YMDH }}.nc4
+      obsfile: $(DATA)/diags/diag_adpsfc_snow.nc4
```

Comment:
Just a note, the current operational convention is to have PDYcyc in the diag filename. Is that something we want to change or continue with? For diags, I would like to keep it if possible. @aerorahul , thoughts?

Reply:
I just copied the files that @jiaruidong2017 pointed me to.
If I revert them here, an update may be needed in the global-workflow.
What do you want me to do in the light that we will move away from having a time-stamp in this file?

Reply:
@aerorahul you can just revert them back and we don't need to make any changes to the global-workflow, because the diagnostic files `diag*.nc4` in `$(DATA)/diags/` will be tarred and gzipped into COM for archive.